core/arch/x86/decode_table.c

```diff
@@ -6559,7 +6563,8 @@ const instr_info_t rm_extensions[][8] = {
   { /* rm extension 2 */
     {OP_swapgs, 0xf80f0177, "swapgs", xx, xx, xx, xx, xx, mrm|o64, x, END_LIST},
     {OP_rdtscp, 0xf90f0177, "rdtscp", edx, eax, xx, xx, xx, mrm|xop, x, exop[10]},/*AMD-only*/
-    {INVALID,   0x0f0131, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {OP_monitorx, 0xfa0f0177, "monitorx",  xx, xx, eax, ecx, edx, mrm, x, END_LIST},/*AMD-only*/
```

Comment:
The other thing is whether the address should be treated as a memory reference or not but let's make that completely orthogonal to this PR.  I filed#4013.  Maybe you could add a comment here like `XXX i#4013: Treat address in xax as IR memref?`.

Reply:
done, added comments above monitor and monitorx